Library/Formula/norm.rb

```diff
@@ -0,0 +1,30 @@
+class Norm < Formula
+  desc "NACK-Oriented Reliable Multicast"
+  homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"
+  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
```

Comment:
The 'b' isn't really beta.  I should probably change my versioning number schema, perhaps changing the 'b' for "build" to an 'r' for "release" instead to reduce this recurring question.  The posted tarballs represent stable release.  I will be posting a new stable release tarball soon that includes the  One thing to note is the current stable release that is posted is version 1.5b4 (the 1.5b3 version you are using is from the "archive" of prior releases), but there have been a couple of bug fixes in the code that may benefit ZeroMQ use.  I may even increment to a 1.6 version number if I drop the 'b' delimiter in favor of something else.